lib/routes.js

```diff
@@ -62,6 +63,17 @@ function poll (req, res, next) {
         }
       }
 
+      let terms = false;
```

Comment:
We can do this without a mutable variable. Make a function that creates the **terms** object, and do:

```
const terms = config.termsScreenActive ? createTerms(config) : null
```